.storybook/main.js

```diff
@@ -42,6 +48,18 @@ module.exports = {
 			],
 		} );
 
+		config.module.rules.push({
+			test: /\.svg$/,
+			loader: 'svg-react-loader',
+		});
+
+		// Leaving this commented to attract the ire of my reviewers,
+		// and hopefully also a suggestion to make this go.
```

Comment:
consider my ire attracted 😄  what does it do though?

Reply:
It's the handy shortcut we have in `qz-react` that lets us use this path:

```
import Icon from `svg/icon`;
```

so our icons can live in `public/svgs` and be shared, instead of being warehoused like this with a single component.

Reply:
1. i'd probably leave them alongside the component unless they were actually reused
2. i wouldn't use an alias, bc then you'd just need to implement the same alias in qz-react and any other project that consumed prism (a confusing requirement)
3. i'd probably put them in `src/svg`. the way we store SVGs in qz-react is pretty odd, where some SVGs are loaded over network and others are inlined. none of the SVGs in prism will be loaded over the network, so "public" doesn't really make sense